README.md

```diff
@@ -24,19 +24,23 @@
 </p>
 
 ## What?
+
 This is a faucet for neo-local. It's made with Node and React and uses your local neo-scan to determine and distribute rewards.
 Everything is setup to be extremely configurable, for more info check the Configurable environment variables below.
 
 ## Features
-* Request NEO/GAS for an existing address
-* Generate a new account (WIF and address will be displayed) and fill it with NEO/GAS
-* Automatic GAS claims for the faucet account
+
+- Request NEO/GAS for an existing address
+- Generate a new account (WIF and address will be displayed) and fill it with NEO/GAS
+- Automatic GAS claims for the faucet account
 
 ## Configurable environment variables
-* `NEOSCAN` is the address of your neo-local NEOSCAN instance. Defaults to `localhost:4000` which is the standard neo-scan URL in neo-local
-* `MIN_BLOCK` is the minimum amount of blocks that need to have passed before reapplying. Defaults to `5000` blocks
-* `FAUCET_WIF` is the WIF of the Faucet account. The standard value is `KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr`, which holds all NEO and GAS in the neo-privatenet image
-* `FAUCET_ADDRESS` is the domain where the faucet is located. Defaults to `localhost`
-* `PORT` is the port where the faucet is located. Defaults to `4002`
-* `NEO_REWARD` is the amount of NEO the faucet rewards. Defaults to `100`
-* `GAS_REWARD` is the amount of GAS the faucet rewards. Defaults to `2000`
+
+- `NEOSCAN` is the address of your neo-local NEOSCAN instance. Defaults to `localhost:4000` which is the standard neo-scan URL in neo-local
```

Comment:
Worth having these in A-Z order to ease readability?

Reply:
Will do this in a separate PR, thanks cap!